opengemini/query.go

```diff
@@ -78,11 +74,8 @@ func (c *client) queryPost(q Query) (*QueryResult, error) {
 	req.queryValues.Add("db", q.Database)
 	req.queryValues.Add("q", q.Command)
 
-	if c.config.Encoding == MSGPACK {
-		if req.header == nil {
-			req.header = make(http.Header)
-		}
-		req.header.Set("Accept", "application/x-msgpack")
+	if c.config.Codec == MSGPACK {
+		setRequestHeaders(&req, c.config.Codec)
```

Comment:
I think setting up a method for `Codec` is a better choice.

Reply:
Can you check  again ?